packages/cli-lib/api/sandboxes-sync.js

```diff
@@ -0,0 +1,37 @@
+const http = require('../http');
+const SANDBOXES_SYNC_API_PATH = 'sandboxes-sync/v1';
+
+// fromHubId is the parent portal here, which is required as the first param. Parent account key can only run the sync
```

Comment:
What is `toHubId` here? I would think it's sandbox Id, but that seems to be represented here as `sandboxHubId`.

Reply:
Ok, reading below in the `cli-lib/sandboxes.js` file, you write that `toHubId` is the `target account for the sync` and that `sandboxHubId` is `sandbox portal ID for the sync`. In what cases will these two IDs differ? I would think they'd always be the same.

Reply:
Yep good catch, its a left over parameter that we never ended up stripping out on our API side that is a requirement still. There may be differences in the future with push to parent functionality, as well as sandbox to sandbox syncs so I think it'd be good to leave like this for now